### DISCUSSION TITLE
请问如何把参考文献中的DOI链接去掉？

### DISCUSSION THREAD

**Discussion by WangKaicgcx** [nju-lug/NJUThesis#78] [Q&A] [answered]
试了\PassOptionsToPackage{doi=false}{biblatex}这个代码，好像还是不行。

**Comment by atxy-blip** [comment, marked as answer]
v0.14.0 之前的版本请在导言区加

```
\ExecuteBibliographyOptions{doi=false}
```

-----

v0.14.0 以后的版本把 `biblatex` 宏包挂在导言区后载入，可在导言区使用

```
\PassOptionsToPackage{doi=false}{biblatex}
```

-----

<https://github.com/nju-lug/NJUThesis/commit/2e06a31c72fb1c89f91a7ad34201c088c0256dfc> 提供了一个相关的选项 `bib/option`，会随 v0.15.0 发布。

**Reply by WangKaicgcx** [reply]
> 
可以了，感谢。

**Comment by ch1zhang** [comment]
@atxy-blip 您好，请问最新的版本中如何把location信息去掉

**Reply by atxy-blip** [reply]
我建议直接把 `bib` 文件里的相关信息删除，就不会打印出来。如果希望有一个通用的改动，给 `biblatex-gb7714-2015` 打补丁，就需要求助 @hushidong 了。

**Comment by hushidong** [comment]
加doi=false选项就行了呀。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
